Contributing.md

```diff
@@ -0,0 +1,212 @@
+* [FAQ](#faq)
+* [More resources](#more-resources)
+
+## Ways to contribute
+
+Here are some ways you can contribute to this documentation:
+
+* To make small changes to an article, [Contribute using GitHub](#contribute-using-github).
+* To make large changes, or changes that involve code, [Contribute using Git](#contribute-using-git).
+* Report documentation bugs via GitHub Issues
+* Request new documentation at the [Office Developer Platform UserVoice](http://officespdev.uservoice.com) site.
+
+## Contribute using GitHub
+
+Use GitHub to contribute to this documentation without having to clone the repo to your desktop. This is the easiest way to create a pull request in this repository. Use this method to make a minor change that doesn't involve code changes. 
+
+**Note** Using this method allows you to contribute to one article at a time.
+
+### To Contribute using GitHub
+
+1. Find the article you want to contribute to on GitHub. 
+
+    If the article is in MSDN, choose the **suggest and submit changes** link in the **Contribute to this content** section and you'll be taken to the same article on GitHub.
+2. Once you are on the article in GitHub, sign in to GitHub (get a free account [Join GitHub](https://github.com/join).
+3. Choose the **pencil icon** (edit the file in your fork of this project) and make your changes in the **<>Edit file** window. 
+4. Scroll to the bottom and enter a description.
+5. Choose **Propose file change**>**Create pull request**.
+
+You now have successfully submitted a pull request. Pull requests are typically reviewed within 10 business days. 
+
+
+## Contribute using Git
+
+Use Git to contribute substantive changes, such as:
+
+* Contributing code.
+* Contributing changes that affect meaning.
+* Contributing large changes to text.
+* Adding new topics.
+
+### To Contribute using Git
+
+1. If you don't have a GitHub account, set one up at [GitHub](https://github.com/join). 
+2. After you have an account, install Git on your computer. Follow the steps in [Setting up Git Tutorial](https://help.github.com/articles/set-up-git/).
+3. To submit a pull request using Git, follow the steps in [Use GitHub, Git, and this repository](#use-github-git-and-this-repository).
+4. You will be asked to sign the Contributor's License Agreement if you are:
+
+    * A member of the Microsoft Open Technologies group.
+    * A contributors who doesn't work for Microsoft.
+
+As a community member, you must sign the Contribution License Agreement (CLA) before you can contribute large submissions to a project. You only need to complete and submit the documentation once. Carefully review the document. You may be required to have your employer sign the document.
+
+Signing the CLA does not grant you rights to commit to the main repository, but it does mean that the Office Developer and Office Developer Content Publishing teams will be able to review and approve your contributions. You will be credited for your submissions.
```

Comment:
Could we change this to the present tense instead of future tense? "will be" -> "are"

Reply:
Done.